pkg/logger/logger.go

```diff
@@ -50,6 +51,18 @@ func GetLogLevel() int {
 	return logLevel
 }
 
+// LogLevelToName takes an int and returns the corresponding log level name
+func LogLevelToName(logLevel int) (string, error) {
+	var err error
+	for k, v := range logLevelMap {
+		if v == logLevel {
+			return k, nil
+		}
+	}
+	err = fmt.Errorf("%s is not a valid log level", strconv.Itoa(logLevel))
```

Comment:
I think just %d instead of %s should work, without needing to use `strconv.Itoa`